### PR TITLE
chore: add logo and frame

### PR DESCRIPTION
**Pull request by davidfurlong**
Adds
- logo
- open graph image
- a simple open frame

### CODE REVIEW

File: vocs.config.tsx [hunk 1]
```diff
@@ -1,9 +1,44 @@
...
+          <meta name="of:accepts:*" content="*" />
+          <meta
+            name="fc:frame:image"
+            content="https://opengraph.xyz/opengraph.png"
+          />
+          <meta
+            property="og:image"
+            content="https://openframes.xyz/opengraph.png"
+          />
+          <meta name="fc:frame:button:1" content="Read Docs" />
```
Comment (stephancill) [inline vocs.config.tsx:31]: action="link" target="https://openframes.xyz"

File: vocs.config.tsx [hunk 2]
```diff
@@ -1,9 +1,44 @@
...
+          <meta property="og:type" content="website" />
+          <meta name="fc:frame" content="vNext" />
+          <meta name="of:version" content="vNext" />
+          <meta name="of:accepts:farcaster" content="vNext" />
+          <meta name="of:accepts:xmtp" content="2024-02-09" />
+          <meta name="of:accepts:lens" content="1.0.0" />
+          <meta name="of:accepts:*" content="*" />
+          <meta
+            name="fc:frame:image"
+            content="https://opengraph.xyz/opengraph.png"
```
Comment (stephancill) [inline vocs.config.tsx]: openframes.xyz

**Review by stephancill — CHANGES_REQUESTED** [review]

File: vocs.config.tsx (hunk 1 above)
Comment (stephancill) [reply vocs.config.tsx:31]: @davidfurlong  the frame button is still broken

Comment (davidfurlong) [reply vocs.config.tsx:31]: are you sure? https://debugger.framesjs.org/?url=https%3A%2F%2Fopenframes.xyz%2F

Comment (stephancill) [reply vocs.config.tsx:31]: realized that it was working yesterday after commenting, my bad

**Pull request merged**